presto-product-tests/bin/run_on_docker.sh

```diff
@@ -52,7 +52,6 @@ function check_presto() {
 }
 
 function run_product_tests() {
-  local REPORT_DIR="${PRODUCT_TESTS_ROOT}/target/test-reports"
```

Comment:
Wasn't this varable working? Why it was moved?

Reply:
It's used just to clear up the test-reports, it's not passed anywhere. I've moved it outside as it's easiest to export it as an env variable and simply use it here.

Reply:
ah right... 

That is because you should modify `presto-product-tests/conf/docker/files/run-tempto.sh`

Reply:
Hm I can't do it in run-tempto.sh or in run-tempto-on-docker.sh because I'm using this REPORT_DIR to mount a different volume for reports path - depending on the tests you want to execute. So I need to have REPORT_DIR known at the moment the $ENVIRONMENT/compose.sh up is executed and it's executed from this script :/

Reply:
Just mount entire target directory. I think it is even already mounted.